### ISSUE TITLE
Add support for setting atifact paths directly

### ISSUE THREAD

**Issue by pmengelbert** [Azure/dalec#112]
Here is an example of a use-case:

Requirements
- We want to build the eraser controller manager image
- We need it to be a drop-in replacement to avoid breaking workflows

The eraser helm chart has the entrypoint hard-coded as `/manager`. This is technically wrong, but it's a problem upstream that we can't fix. Dalec doesn't have a way to specify the installation path outside of a) `/usr/bin` or `/usr/<subpath(s)>/bin`.

Even though it's against standards, to avoid breaking downstream consumers' workflows, we need to allow the entrypoint to exist at `/manager`. This is obviously unacceptable for a package but is pretty standard practice for a container image.

One proposed solution is adding a step to add symlinks in the final image (after the package has been installed)

**Comment by cpuguy83** [comment]
I don't think we should should support setting the artifact path directly as what happens with the artifact is dependent on the backing implementation.
For rpms we install to /usr/bin because that's what you are supposed to do (at least for system packages, which is what we are building as for our own stuff).

Adding support for symlinking in the container is fine.
Alternatively support for post-install scripts could deal with this as well, though technically I think that's not the best way to go about achieving the goal of having a drop-in container replacement.
